specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -2760,6 +2795,29 @@
       },
       "description": "The gallery artifact version source."
     },
+    "ExecutedValidation": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "This property specifies the type of image version validation."
+        },
+        "status": {
+          "type": "string",
+          "description": "This property specifies the status of the validationProfile of the image version."
```

Comment:
consider updating to enums if applicable

Reply:
which enum? do you mean to add in the status?